framework/dbt/basic/macros/overrides.sql

```diff
@@ -1,116 +0,0 @@
-{% macro default__reset_csv_table(model, full_refresh, old_relation, agate_table) %}
-    {% set sql = "" %}
-    {% if full_refresh %}
-        {{ adapter.drop_relation(old_relation) }}
-        {% set sql = create_csv_table(model, agate_table) %}
-    {% else %}
-        {{ adapter.truncate_relation(old_relation) }}
-        {% set sql = "delete from " ~ old_relation %}
-    {% endif %}
-
-    {{ return(sql) }}
-{% endmacro %}
-
-{% macro generate_schema_name(custom_schema_name, node) -%}
-
-    {%- set default_schema = target.schema -%}
-    {%- if custom_schema_name is none -%}
-
-        {{ default_schema }}
-
-    {%- else -%}
-
-        {{ custom_schema_name | trim }}
-
-    {%- endif -%}
-
-{%- endmacro %}
-
```

Comment:
Also refactored, but into the documentation instead.

- https://github.com/crate/cratedb-guide/pull/151